aal_purchase_form/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'aal_Purchase_Form',
+    'version': '12.0.1.0.1',
+    'author': 'Ecosoft',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/ecosoft-odoo/aal',
+    'category': 'Report',
+    'depends': [
+        'web',
+        'purchase',
+        'purchase_order_secondary_unit',
```

Comment:
```suggestion
        'purchase_order_secondary_unit',
```
AAL used this module?